### PR TITLE
docs: update install guide for angular elements

### PR DESCRIPTION
**Pull request by bampakoa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]

File: aio/content/guide/elements.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ The recently-developed [custom elements](https://developer.mozilla.org/en-US/doc
 </tr>
 </table>
 
-Use the [Angular CLI](cli) to add the `@angular/elements` package to your workspace.
+To add `@angular/elements` package to your workspace, run the following command:
```
Comment (gkalpak) [inline aio/content/guide/elements.md:126]:
```suggestion
To add the `@angular/elements` package to your workspace, run the following command:
```

**Review by gkalpak — APPROVED** [review]
Thx, @bampakoa 🙇

One minor comment. Otherwise lgtm 🚀

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by atscott** [comment]
This PR was merged into the repository by commit 22edf0263fbce22cb5d2ab7f2fd4094fc8db1885.

**Pull request closed without merging**